doc/specs/collages.md

```diff
@@ -0,0 +1,140 @@
+# Collages
+
+## Table of contects
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+The collage system is an important feature.
+It provides an orthogonal way to either artist page or tagging system to organise Album, based (mostly) on Facts.
```

Comment:
What is a "Fact"? Collages are mostly based on a theme, be it someone preferences, or the Top 100 albums of the year of some magazine.